app/experimenter/templates/experiments/rapid.html

```diff
@@ -2,7 +2,15 @@
 {% load static%}
 
 
+{% load static %}
```

Comment:
This is called above on L2. Does it need to be called twice? Sorry, I'm not totally certain what these template shortcodes are doin.

Reply:
Oh good catch! Yeah you shouldn't need to do that twice.